backends/etcdv3/client.go

```diff
@@ -119,7 +119,7 @@ func (c *Client) WatchPrefix(prefix string, keys []string, waitIndex uint64, sto
 
 	for wresp := range rch {
 		for _, ev := range wresp.Events {
-			fmt.Println(string(ev.Kv.Key))
+			log.Debugf("Key updated %s", string(ev.Kv.Key))
```

Comment:
There is no such method as `Debugf`, please change in to `Debug`

Reply:
embarrassing...